config/pmd.xml

```diff
@@ -83,6 +84,8 @@
     <!-- <exclude name="CommentRequired"/> -->
     <!-- we use class comments as source for xdoc files, so content is big and that is by design -->
     <exclude name="CommentSize"/>
+    <!-- Pollutes code with comments -->
+    <exclude name="CommentDefaultAccessModifier"/>
```

Comment:
Please share in gist output from this rule fro review.
We should not use default at all.
If there some unfortunate cases, we need issue on this.
We will keep a rule active to highlight such cases.

Again for migration time, might be better to keep suppress with reference to new issue for this in our tracker

Reply:
It complains on every non-public non-private method/ctor:
https://gist.github.com/pbludov/01574efeda97b8532cb09db3cf393e86

Reply:
please move this to separate issue ,  and suppress till that new issue.
Most of such code need to be fixed.

Reply:
Done https://github.com/checkstyle/checkstyle/issues/5665